examples/nlp/token_classification/token_classification.py

```diff
@@ -48,7 +48,7 @@
 parser.add_argument("--ignore_start_end", action='store_false')
 parser.add_argument("--ignore_extra_tokens", action='store_false')
 parser.add_argument("--none_label", default='O', type=str)
-parser.add_argument("--shuffle_data", action='store_false')
+parser.add_argument("--shuffle_data", default=True, type=bool)
```

Comment:
This will always resolve to True, even if you do --shuffle_data=False. To use boolean, please you action=....

Reply:
wrong call: you call your script with --shuffle_data or without it